src/nvim/po/zh_TW.UTF-8.po

```diff
@@ -814,7 +814,7 @@ msgstr "E227: %s 的 mapping 已經存在"
 
 #: ../eval.c:8692
 msgid "extend() argument"
-msgstr ""
+msgstr "--cmd 參數"
```

Comment:
that doesn't look right...

Reply:
yup don't know who translate it in zh_CN.. I copy it.. need retranslate